pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/PulsarAdminImpl.java

```diff
@@ -215,13 +212,35 @@ public PulsarAdminImpl(String serviceUrl,
         }
     }
 
+    private Authentication determineAuth(ClientConfigurationData conf) throws PulsarClientException {
```

Comment:
This always return a non-null auth which allows the calling code to not deal w/ that. 

Note that `conf.getAuthentication` itself is null-proof and will return `AuthenticationDisabled` if so.